### ISSUE TITLE
Crash on mousing over item, 1.19.4 Quilt, with Sodium.

### ISSUE THREAD

**Issue by griffin4cats** [AHilyard/LegendaryTooltips#79]
### Steps to reproduce
1. Create new 1.19.4 instance
2. Install Quilt loader 0.18.10 
3. Install the following mods:
- LegendaryTooltips 1.4.3
- Sodium 0.4.10
- Iceberg 1.1.9 (or 1.1.8)
- Prism 1.0.4
- Forge Config Api port 6.0.2
- QFAPI 6.0.0-beta-9
4. Create new world or connect to server
5. Mouse over a bow while in inventory

Game crashes immediately.
This issue is not present when Sodium and/or LegendaryTooltips are missing, indicating a direct incompatibility between Sodium and LegendaryTooltips.

[Log file included here](https://mclo.gs/LIsj5P2)

I would also like to take the time to ask for Iceberg version 1.1.10 to be removed from Modrinth, and version 1.1.9 to be uploaded instead, as it made finding version 1.1.9 harder, but this exact issue is outside of the scope of this issue report.

All of these files were downloaded from Modrinth, except Iceberg 1.1.9 which was downloaded from Curseforge. The 1.1.8 version from modrinth was tested as well, it fails similarly.

**Comment by griffin4cats** [comment]
I tested this in Iceberg 1.1.10 as well, and the same crash occurred. 
I believe this may be related to [this issue with iceberg](https://github.com/AHilyard/Iceberg/issues/26). That issue was fixed in Iceberg 1.1.10, but Iceberg 1.1.10 has been retracted (though not retracted on modrinth yet) because it's only compatible with Sodium 0.4.11, which was also retracted.

As such there is currently no way to use LegendaryTooltips and Sodium at the same time on Quilt right now.
I marked this as a LegendaryTooltips issue because that's what it appeared to me to be, but this may be an Iceberg issue instead.
